host-interaction/process/inject/process-ghostly-hollowing.yml

```diff
@@ -0,0 +1,26 @@
+rule:
+  meta:
+    name: process ghostly hollowing
+    namespace: host-interaction/process/inject
+    authors:
+      - sara.rincon@mandiant.com
+    scopes:
+      static: function
+      dynamic: call
+    references:
+      - https://github.com/hasherezade/transacted_hollowing/tree/main#ghostly-hollowing
+    examples:
+      - 3b2eba4789bd4a799fe18476a4d1ce9f37ecc4c202eb406e06425c7e792904ff:0x140007aa0
+      - 3b2eba4789bd4a799fe18476a4d1ce9f37ecc4c202eb406e06425c7e792904ff:0x140007840
```

Comment:
is one example enough here or are the two functions different and both meaningful?